tests/integration/api/test_api.py

```diff
@@ -521,221 +357,156 @@ def _compare_screenboard(apiBoard, expectedBoard):
-
-        # dog.Monitor.unmute(monitor_id, scope='host:foo')
-        # monitor = dog.Monitor.get(monitor_id)
-        # eq(monitor['options']['silenced'], {})
-
-        # options = {
-        #     "silenced": {"host:abcd1234": None, "host:abcd1235": None}
-        # }
-        # dog.Monitor.update(monitor_id, query=query, options=options)
-        # monitor = dog.Monitor.get(monitor_id)
-        # eq(monitor['options']['silenced'], options['silenced'])
-
-        # dog.Monitor.unmute(monitor_id, all_scopes=True)
-        # monitor = dog.Monitor.get(monitor_id)
-        # eq(monitor['options']['silenced'], {})
-
-        # dog.Monitor.delete(monitor_id)
-
-    @pytest.mark.monitor
     def test_downtime(self):
         start = int(time.time())
         end = start + 1000
 
         # Create downtime
-        downtime_id = dog.Downtime.create(scope='env:staging', start=start, end=end)['id']
-        dt = dog.Downtime.get(downtime_id)
-        eq(dt['start'], start)
-        eq(dt['end'], end)
-        eq(dt['scope'], ['env:staging'])
-        eq(dt['disabled'], False)
+        downtime = dog.Downtime.create(scope="env:staging", start=start, end=end)
+        assert downtime["start"] == start
+        assert downtime["end"] == end
+        assert downtime["scope"] == ["env:staging"]
+        assert downtime["disabled"] is False
 
         # Update downtime
         message = "Doing some testing on staging."
         end = int(time.time()) + 60000
-        dog.Downtime.update(downtime_id, scope='env:staging',
-                            end=end, message=message)
-        dt = dog.Downtime.get(downtime_id)
-        eq(dt['end'], end)
-        eq(dt['message'], message)
-        eq(dt['disabled'], False)
+        downtime = dog.Downtime.update(downtime["id"], scope="env:staging", end=end, message=message)
+        assert downtime["end"] == end
+        assert downtime["message"] == message
+        assert downtime["disabled"] is False
 
         # Delete downtime
-        dog.Downtime.delete(downtime_id)
-        dt = dog.Downtime.get(downtime_id)
-        eq(dt['disabled'], True)
+        assert dog.Downtime.delete(downtime["id"]) is None
+        downtime = dog.Downtime.get(downtime["id"])
+        assert downtime["disabled"] is True
 
-    @pytest.mark.monitor
     def test_service_check(self):
-        dog.ServiceCheck.check(
-            check='check_pg', host_name='host0', status=1,
-            message='PG is WARNING', tags=['db:prod_data'])
+        assert dog.ServiceCheck.check(
+            check="check_pg",
+            host_name="host0",
+            status=1,
+            message="PG is WARNING",
+            tags=["db:prod_data"]
+        ) == {"status": "ok"}
 
-    @pytest.mark.host
     def test_host_muting(self):
-        hostname = "my.test.host"
-        message = "Muting this host for a test."
         end = int(time.time()) + 60 * 60
+        hostname = "my.test.host" + str(end)
```

Comment:
Similar to whats done for the user, would it make sense to put a random token in here so concurrent test runs don't impact one another?

Reply:
The timestamp is the "random" key. Having two tests run at the exact same second would be really unlucky I assume, and shouldn't happen.

Reply:
missed the `+ str(end)`. Thanks for the confirmation